### ISSUE TITLE
False positives in I18nSniff

### ISSUE THREAD

**Issue by swissspidy** [WordPress-Coding-Standards/WordPress-Coding-Standards#1418]
In my [Preferred Languages](https://wordpress.org/plugins/preferred-languages/) plugin I am using some low-level WordPress functions to load some text domains.

One piece of code in it looks like this:

```php
$translations = get_translations_for_domain( $domain );
$translation  = $translations->translate( $text );
```

Note: `get_translations_for_domain()` returns an instance of the [`Translations`](https://developer.wordpress.org/reference/classes/translations/) class. Its `translate()` method takes only one argument, the text to translate.

This code results in the following PHPCS output:

```
FOUND 2 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------------------------------------
 2 | WARNING | Use of the "translate()" function is reserved for low-level API usage. (WordPress.WP.I18n.LowLevelTranslationFunction)
 2 | ERROR   | Missing $domain arg. (WordPress.WP.I18n.MissingArgDomain)
 2 | ERROR   | The $text arg must be a single string literal, not "$text". (WordPress.WP.I18n.NonSingularStringLiteralText)
----------------------------------------------------------------------------------------------------------------------------------------
```

I think the main issue here is that the sniff doesn't distinguish between `translate()` as a function and the method I am calling on this object. If it would do this, the errors and warnings would go all away, I think.

----

I'm using PHP 7.2, WPCS 0.14 (current master), and PHPCS 3.3.0

**Comment by jrfnl** [comment]
Closing as duplicate of #1266 which has already been fixed. The fix will be included in the upcoming 1.0.0 release.

**Comment by swissspidy** [comment]
Oh, I didn‘t see that one. Thanks for letting me know & already fixing it!
